.build-tools/pkg/metadataschema/builtin-authentication-profiles.go

```diff
@@ -20,6 +20,61 @@ import (
 // ParseBuiltinAuthenticationProfile returns an AuthenticationProfile(s) from a given BuiltinAuthenticationProfile.
 func ParseBuiltinAuthenticationProfile(bi BuiltinAuthenticationProfile) ([]AuthenticationProfile, error) {
 	switch bi.Name {
+	case "aws":
+		return []AuthenticationProfile{
+			{
+				Title:       "AWS: Access Key ID and Secret Access Key",
+				Description: "Authenticate using an Access Key ID and Secret Access Key included in the metadata",
+				Metadata: []Metadata{
+					{
+						Name:        "accessKey",
+						Required:    true,
+						Sensitive:   true,
+						Description: "AWS access key associated with an IAM account",
+						Example:     `"AKIAIOSFODNN7EXAMPLE"`,
+					},
+					{
+						Name:        "secretKey",
+						Required:    true,
+						Sensitive:   true,
+						Description: "The secret key associated with the access key",
+						Example:     `"wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"`,
+					},
+				},
+			},
+			{
+				Title:       "AWS: Credentials from Environment Variables",
+				Description: "Use AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY from the environment",
+				Metadata:    []Metadata{},
+			},
+		}, nil
+	case "gcp":
+		return []AuthenticationProfile{
+			{
+				Title:       "GCP: Use private_key_id and private_key",
+				Description: "Environment variable to provide the location of a credential JSON file",
+				Metadata: []Metadata{
+					{
+						Name:        "private_key_id",
+						Required:    true,
+						Description: "The ID of the private key from the credentials",
+						Example:     `"privateKeyId"`,
+					},
+					{
+						Name:        "private_key",
+						Required:    true,
+						Sensitive:   true,
+						Description: "GCP credentials private key. Replace with x509 cert",
```

Comment:
I left the same comment on another PR.

Is this a private key? Or a certificate? They are separate things.

Also, what about these other options? Aren't they auth related too?

<img width="971" alt="image" src="https://github.com/dapr/components-contrib/assets/43508/5fd4aa07-8ca5-455c-9649-64e2cde439b6">

To make things worse, seems that some components use kebab_case and some use camelCase (which is the more appropriate for Dapr):

<img width="980" alt="image" src="https://github.com/dapr/components-contrib/assets/43508/d435cbef-5a73-408e-90b8-595715ab858d">